### PR TITLE
[skip ci] Build img

### PR DESCRIPTION
**Pull request by leseb**

### CODE REVIEW

**Comment by leseb** [comment]
built on top of #905

File: maint-lib/flavors-modified-vs-master.py [hunk 1]
```diff
@@ -17,8 +22,7 @@ def _stage_flavor(flavor):
     staging_dir_pattern = re.compile(r'^\s*STAGING_DIR\s*:\s+(.*)$', re.MULTILINE)
     match = staging_dir_pattern.search(stage_output)
     if not match or not match[1] or match[1] == '':
-        print("Could not find staging dir for:\n{}".format(stage_output))
-        sys.exit(1)
+        fatal("Could not find staging dir for:\n{}".format(stage_output))
```
Comment (ErwanAliasr1) [inline maint-lib/flavors-modified-vs-master.py]: _fatal()

File: maint-lib/flavors-modified-vs-master.py [hunk 2]
```diff
@@ -40,8 +44,7 @@ def _filediff_intersects_sources(filediff, sources):
 try:
     filediff = _run_cmd(['git', 'diff', '--name-only', VS_BRANCH])
 except subprocess.CalledProcessError as c:
-    print('Could not get file diff. Is the VS_BRANCH specified a real branch?')
-    sys.exit(1)
+    fatal('Could not get file diff. Is the VS_BRANCH specified a real branch?')
```
Comment (ErwanAliasr1) [inline maint-lib/flavors-modified-vs-master.py]: _fatal()

File: contrib/build-push-ceph-container-imgs.sh [hunk 3]
```diff
@@ -17,9 +17,17 @@ function login_docker_hub {
   docker login -u "$DOCKER_HUB_USERNAME" -p "$DOCKER_HUB_PASSWORD"
 }
 
+function create_point_release {
+  local latest_tag
+  latest_tag=$(git describe --exact-match HEAD --tags --long 2>/dev/null)
+  if [ "$?" -eq 0 ]; then
+    POINT_RELEASE=$latest_tag
```
Comment (ErwanAliasr1) [inline contrib/build-push-ceph-container-imgs.sh]: commit RELEASE as per https://github.com/ceph/ceph-container/pull/923/commits/f44885956680f8eb717e0973348ddb74b1b677c3

**Comment by ErwanAliasr1** [comment]
As seen with @leseb , commit c43f23f6e1de73e3f692794b7e10b7d6be11b335 is cherry picked as https://github.com/ceph/ceph-container/pull/927/commits/d0d773ea198a8e0262eae075d1a3b8456714d8c1 for the final integration.

Please rebase against #927

**Comment by ErwanAliasr1** [comment]
Please rebase against master as PR #927 got merged.
If you need help to understand how to update your PR against the new scheme, feel free to ping me.

**Pull request closed without merging**